pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -339,6 +339,53 @@ func TestEtcdUpdate(t *testing.T) {
 
 }
 
+func TestNoOpUpdates(t *testing.T) {
+	server, registry := NewTestGenericEtcdRegistry(t)
```

Comment:
Got it - makes sense (although in that case no-op update is not no-op so it will be performed). But having such test makes sense.